session9/src/homework/ex9.java

```diff
@@ -0,0 +1,14 @@
+package homework;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+public class ex9 {
+    public static void main(String[] args) {
+        LocalDate today = LocalDate.now();
+        LocalDate yesterday = LocalDate.of(2024, 1, 1);
+        long calculateDays = ChronoUnit.DAYS.between(today, yesterday);
+        long a = Math.abs(calculateDays);
```

Comment:
Niceee! The only thing to mention is that this code should be part of `daysBetween` method.